console/jsre/deps/web3.js

```diff
@@ -2645,6 +2646,7 @@ function Web3 (provider) {
     this._requestManager = new RequestManager(provider);
     this.currentProvider = provider;
     this.klay = new Klay(this);
+	this.eth = new Eth(this);
```

Comment:
Wrong indentation?

Reply:
I added commit to fix this :) THank yoU!